.github/workflows/release.yml

```diff
@@ -0,0 +1,32 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  release:
+    name: Trigger a release
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+        with:
+          ref: master
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+      # This step will retry until required check passes
+      # and will fail the whole workflow if the check conclusion is not a success
+      # https://github.com/marketplace/actions/wait-on-check
+      - name: Wait on checks
+        uses: lewagon/wait-on-check-action@v0.1
+        with:
+          ref: master
+          check-name: depcheck # name of the last existing check for push
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          wait-interval: 20 # seconds
+      - name: Create tag
+        if: ${{ success() }}
+        run: node ../index.js tag --github -v
```

Comment:
you can specify the [`working-directory`](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepsrun) to perform node script execution without resorting to relevant path operations

Reply:
I misunderstood it, I thought the workflow directory would run commands from its own directory

But apparently, the workflow considers the root directory to run anything. So no need for directory declaration at all